File: Cpp/Problems/problem1.cpp
//1
// to check if given character is in lower case or uppercase
#include<iostream>
int main(int argc,char** argv){
  char ch;
  std::cin>>ch;
  if(ch>='a'&&ch<='z'){
    std::cout<<"This is lowerCase";

  }else if(ch>='A'&&ch<='Z'){
    std::cout<<"This is upperCase";
  }else{
    std::cout<<"This is numeric";
  }
  std::cout<<" ";
  return 0;
}

File: Rust/RustTutorials/Problems/problem1.rs
//To check if given character is in lower case or uppercase
use std::io;

fn main() {
    println!("Program to check if character is uppercase of lowercase");
    let mut console_read = String::new();
    io::stdin()
        .read_line(&mut console_read)
        .expect("Failed to read");
    let ch: char = console_read.trim().parse().expect("Failed to Parse");
    if ch >= 'a' && ch <= 'z' {
        print!("This is lowerCase");
    } else if ch >= 'A' && ch <= 'Z' {
        print!("This is upperCase");
    } else {
        print!("Not a character");
    }
    return ();
}
